hw/top_earlgrey/dv/verilator/chip_sim.core

```diff
@@ -105,7 +105,11 @@ targets:
           - '--trace-max-array 1024'
           - '--unroll-count 512'
           - '-CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DVL_USER_STOP -DTOPLEVEL_NAME=chip_sim_tb"'
+          # TODO: Variable expansion depends on edalize internals. Find better solution.
+          #       (Applies to LDFLAGS expansion below as well)
+          - '-CFLAGS "$(CFLAGS_FOR_BUILD)"'
           - '-LDFLAGS "-pthread -lutil -lelf"'
+          - '-LDFLAGS "$(LDFLAGS_FOR_BUILD)"'
```

Comment:
Ah not quite. CI is failing with 

> verilator -f lowrisc_dv_chip_verilator_sim_0.1.vc --trace --trace-fst --trace-structs --trace-params --trace-max-array 1024 --unroll-count 512 -CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DVL_USER_STOP -DTOPLEVEL_NAME=chip_sim_tb" -CFLAGS "" -LDFLAGS "-pthread -lutil -lelf" -LDFLAGS "" -Wall --threads 4 -Wno-fatal --threads 4
> %Error: Invalid option: -pthread -lutil -lelf

We probably need to pass CFLAGS and LDFLAGS only once, like this:

```suggestion
          # TODO: Variable expansion depends on edalize internals. Find better solution.
          #       (Applies to LDFLAGS expansion below as well)
          - '-CFLAGS "$(CFLAGS_FOR_BUILD) -std=c++11 -Wall -DVM_TRACE_FMT_FST -DVL_USER_STOP -DTOPLEVEL_NAME=chip_sim_tb"'
          - '-LDFLAGS "$(LDFLAGS_FOR_BUILD) -pthread -lutil -lelf"'
```

Reply:
Sounds good.  To confirm, that still works on our side also.